mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -224,6 +242,42 @@ def _load_mesh(runStreams):  # {{{
         refTopDepth, refLayerThickness  # }}}
 
 
+def _cache_velocity_climatologies(config, startDateClimo, endDateClimo,
+                                  inputFilesClimo, simulationStartTime,
+                                  variableMap, calendar):  # {{{
+    '''compute yearly velocity climatologies and cache them'''
+
+    variableList = ['avgNormalVelocity',
+                    'avgVertVelocityTop']
+
+    outputDirectory = build_config_full_path(config, 'output',
+                                             'mpasClimatologySubdirectory')
+
+    try:
+        os.makedirs(outputDirectory)
+    except OSError:
+        pass
+
+    ds = open_multifile_dataset(fileNames=inputFilesClimo,
+                                calendar=calendar,
+                                config=config,
+                                simulationStartTime=simulationStartTime,
+                                timeVariableName='Time',
+                                variableList=variableList,
+                                variableMap=variableMap,
+                                startDate=startDateClimo,
+                                endDate=endDateClimo)
+
+    climatology.update_start_end_year(ds, config, calendar)
```

Comment:
I personally find this `module.method` syntax a little confusing.  It is ok here, but I'd prefer we put a comment here stating so that it is obvious that `climatology` isn't a local instance of a class. Maybe something like

` # compute the starting and ending dates for the climatology`